structs/student_validator.c

```diff
@@ -0,0 +1,46 @@
+#include<stdio.h>
+
+typedef struct 
+{
+    int age;
+    int birth;
+    char name[10];
+} StudentDate;
+
+int checkCard(StudentDate card) 
+{
+    int age = card.age;
+    int birth = card.birth;
+
+    return age < 18 || birth > 1991;
+}
+
+StudentDate feedBack(StudentDate card)
+{
+    int age = card.age;
+    int birth = card.birth;
+
+    if(checkCard(card))
+    {
+        printf("is not for you");
+    } else {
```

Comment:
`else` should be placed on a separate line between `}` and `{` braces

Reply:
Done.
 if (checkCard(card))
    {
        printf("Sorry, not for you %d", age);
    } 
    else 
    {
        printf("Nice... \nAge is: %d", age);
    }